types/api/config.d.ts

```diff
@@ -137,6 +137,9 @@ interface ClientConfig {
     IosAppDownloadLink: string;
     IosLatestVersion: string;
     IosMinVersion: string;
+    IntuneMAMEnabled?: string;
```

Comment:
Could this be like this?

```suggestion
    IntuneMAMEnabled?: 'true' | 'false';
```

Reply:
In theory it should be ok